ipywidgets/static/widgets/js/widget_selectioncontainer.js

```diff
@@ -15,7 +15,7 @@ define([
             this.containers = [];
             this.model_containers = {};
             this.children_views = new widget.ViewList(this.add_child_view, this.remove_child_view, this);
-            this.listenTo(this.model, 'change:children', function(model, value) {
+            this.listenTo(this.model, 'change:children', function(model, value, options) {
```

Comment:
Not sure, I think I was using it elsewhere and accidentally added it here.  Shouldn't cause any harm though.